SemiconductorTestLibrary.Extensions/source/InstrumentAbstraction/DCPower/Types.cs

```diff
@@ -150,6 +150,13 @@ public class DCPowerWaveformAcquisitionSettings
         /// The measure trigger type.
         /// </summary>
         public DCPowerMeasureTriggerType MeasureTriggerType { get; set; }
+
+        /// <summary>
+        /// Configures DCPower waveform acquisition settings.
```

Comment:
Please update the wording in a similar way as AOFunctionGenerationSettings.

Reply:
Updated